src/provider/view/components/CircleButton/CircleButton.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+
+import './CircleButton.scss';
+
+type IconType = 'accept' | 'cancel' | 'close' | 'hide' | 'pin';
```

Comment:
I think we typically use a string enum for things like this, rather than a type definition?

Same for `size`

Reply:
Switched to enums.